src/general/world_effects/WorldEffectVisuals.cs

```diff
@@ -1,10 +1,45 @@
-﻿public enum WorldEffectVisuals
+﻿using System.Collections.Generic;
+
+public class WorldEffectVisuals
 {
-    /// <summary>
-    ///   Used as a placeholder value instead of null
-    /// </summary>
-    None,
+    public static readonly Dictionary<WorldEffectTypes, string> EventsIcons = new()
+    {
+        [WorldEffectTypes.UnderwaterVentEruption] = "res://assets/textures/gui/bevel/Eruption.svg",
+        [WorldEffectTypes.GlobalGlaciation] = "res://assets/textures/gui/bevel/GlobalGlaciation.svg",
+        [WorldEffectTypes.MeteorPlainImpact] = "res://assets/textures/gui/bevel/MeteorPlain.svg",
+        [WorldEffectTypes.MeteorIronImpact] = "res://assets/textures/gui/bevel/MeteorIron.svg",
+        [WorldEffectTypes.MeteorPhosphatesImpact] = "res://assets/textures/gui/bevel/MeteorPhosphates.svg",
+        [WorldEffectTypes.MeteorRadioactiveImpact] = "res://assets/textures/gui/bevel/MeteorRadioavtive.svg",
+        [WorldEffectTypes.MeteorGlucoseImpact] = "res://assets/textures/gui/bevel/MeteorGlucose.svg",
+        [WorldEffectTypes.MeteorSulfurImpact] = "res://assets/textures/gui/bevel/MeteorSulfur.svg",
+    };
+
+    public static readonly Dictionary<WorldEffectTypes, string> EventsTooltips = new()
+    {
+        [WorldEffectTypes.UnderwaterVentEruption] = "EVENT_ERUPTION_TOOLTIP",
```

Comment:
This kind of pattern can't be picked up by the localization tool. So something else is needed. It almost seems like it'd be worth it to have a separate world events JSON file that would specify the tooltips, icon paths, *and* the meteors. That way this PR would add just one new JSON info file (which could be used for environmental events in general).

Reply:
What if I just use Localization.Translate() here instead?

Reply:
When will you trigger re-translation then? If the user changes language after the game is started then the old text will stay here. I suppose having `LocalizedString` here could be a viable alternative.